### DISCUSSION TITLE
Permitir asset en las opciones de dropdown selector

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1031] [Proposals]
Hola!
Nos surge la necesidad de crear un selector de color para el CMS pero el componente selector actual no permite incluir iconos ni imágenes en las opciones del dropdown. ¿Creéis que sería factible incluir la esta posibilidad?

<img width="459" alt="image" src="https://user-images.githubusercontent.com/95479574/215474700-531ff3fc-bc4a-42d8-84c5-3170fee5cb84.png">

**Comment by aweell** [comment]
Hola @MontseTelefonica!

Podría haber varias soluciones a esta necesidad intentando modificar el select actual o hacer uno custom.

### 1. Utilizar text input + menu

Se puede utilizar el render y el componente menu para montar una solución atómica. Contra, en el value del text input nunca va a poder verse el color seleccionado.

[Ejemplo →](https://rebrand.ly/3sir305)

### 2. Utilizar el render del radio-group

Otro ejemplo con el render, pero en este caso para unas chips de selección excluyente.

[Ejemplo →](https://rebrand.ly/xkupkdk)

**Nota** Esto próximamente será posible en Mística sin necesidad de usar una chip custom el render del radio

### 3. Utilizar un circle fuera del select y el que tenemos actualmente

[Ejemplo →](https://rebrand.ly/lawz8gt)

En el circle se pintaría el color y el select sería uno como los que tenemos actualmente (solo texto)

**Comment by MontseTelefonica** [comment]
La opción 1 es la que mejor escalabilidad tiene, el único pero que encuentro es que una vez seleccionado el color, yo lo previsualizas en la bolita. ¿Eso no es posible, no?

**Comment by yceballost** [comment]
Vamos a intentar encajar esta discussion en esta issue #1663
